src/header/header.scss

```diff
@@ -415,7 +415,7 @@
       box-shadow: 0 $iui-xxs 0 0 rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-4));
     }
 
-    .iui-label {
+    .iui-button-label {
```

Comment:
Can you also change the class below from iui-icon to iui-button-icon?
![image](https://user-images.githubusercontent.com/9084735/145646033-f5f5856b-8f4b-47b1-bc01-5e686b5ff9b5.png)

Reply:
Done.